packages/x-date-pickers-pro/src/tests/describeRangeValidation/testTextFieldRangeValidation.tsx

```diff
@@ -27,7 +27,7 @@ export const testTextFieldRangeValidation: DescribeRangeValidationTestSuite = (
 ) => {
   const { componentFamily, render, isSingleInput, withDate, withTime } = getOptions();
 
-  if (!['legacy-picker', 'picker'].includes(componentFamily)) {
+  if (!['legacy-picker', 'new-picker'].includes(componentFamily)) {
```

Comment:
This seems to have been an oversight typo, which prevented quite a lot of tests from running 🙈